<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <listeners>
    <listener class-name="shravang.TestComponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="Error Handling Test(failed)" verbose="0">
    <classes>
      <class name="shravang.Tests.ErrorHandlingTest">
        <methods>
          <include name="launchApplication"/>
          <include name="ProductErrorValidation"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- shravang.Tests.ErrorHandlingTest -->
    </classes>
  </test> <!-- Error Handling Test(failed) -->
  <test thread-count="5" parallel="tests" name="StandAlone Test(failed)" verbose="0">
    <classes>
      <class name="shravang.Tests.StandAloneTest">
        <methods>
          <include name="submitOrder" invocation-numbers="0 1"/>
          <include name="launchApplication"/>
          <include name="OrderHistoryTest"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- shravang.Tests.StandAloneTest -->
    </classes>
  </test> <!-- StandAlone Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
